bigtable/client/table.cc

```diff
@@ -45,9 +45,9 @@ namespace {
 
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
-// Call the `google.bigtable.v2.Bigtable.MutateRow` RPC repeatedly until
-// successful, or until the policies in effect tell us to stop.
-void Table::Apply(SingleRowMutation&& mut) {
+namespace noex {
+std::vector<FailedMutation> Table::Apply(SingleRowMutation&& mut,
```

Comment:
This function flags errors in two different ways, it returns a non-empty vector on error, which includes a status, and returns non-OK `grpc::Status`.  Maybe just one of the two?  I think the FailedMutation can be made to work?

Reply:
I agree we can use failed mutation to get status, do you want to do same thing for BulkApply too?

Reply:
Looks like status code in failed mutation for BulkApply doesnt work correctly. Do you want to fix BulkApply to return failed mutation and correct status code or use more of less same signature for Apply for consistency?